packages/odie-client/src/components/message/user-message.tsx

```diff
@@ -66,12 +66,7 @@ export const UserMessage = ( {
 	};
 
 	const renderExtraContactOptions = () => {
-		const currentMessageIndex = chat.messages.findIndex(
-			( msg ) => msg.message_id === message.message_id
-		);
-		const isLastMessage = currentMessageIndex === chat.messages.length - 1;
-
-		return isLastMessage && <GetSupport onClickAdditionalEvent={ handleContactSupportClick } />;
+		return <GetSupport onClickAdditionalEvent={ handleContactSupportClick } />;
```

Comment:
I'm a little worried about what would happen if the user has an ongoing conversation with HEs and the button gets clicked again. I would try to show it, like you did here, but hide when we're already talking with human support.

Reply:
I've added code to hide for chat providers different than 'odie'